### ISSUE TITLE
It works good! but here is a subjection!

### ISSUE THREAD

**Issue by Kingkharun101** [Hello345928/Luna-Grabber#1]
could u change the code so it automatically attaches once u open up ur Laptop it injects the script and opens up so u can easily access the logger without going thru the hassle of looking for it in ur files?

**Comment by Hello345928** [comment]
Yeah i will try my best, i got u. i'll see if i can some way change the code to attach/inject its self using a automated bot in a few days! if not , i will give u an update!
